administrator-manual/en/firewall.rst

```diff
@@ -39,18 +46,22 @@ When a network packet passes through a firewall zone, the system evaluates a lis
 traffic should be blocked or allowed. 
 :dfn:`Policies` are the default rules to be applied when the network traffic does not match any existing criteria.
 
-The firewall implements two default policies editable from the page :menuselection:`Firewall rules` -> :guilabel:`Configure`:
+The firewall implements two default policies:
 
 * :dfn:`Allowed`: all traffic from green to red is allowed
 * :dfn:`Blocked`: all traffic from green to red network is blocked. Specific traffic must be allowed with custom rules.
 
+To change the default policy for Internet access, enable or disable the :guilabel:`Traffic to Internet (red interface)` option from the :guilabel:`Settings` page.
+Policies can be changed also by creating specific rules between zones from :guilabel:`Rules` page.
```

Comment:
```suggestion
Policies can be changed also by creating specific rules between zones from the :guilabel:`Rules` page.
```

Reply:
`the` it's not really needed here, but ok